cmd/test_stores/fake_refinery.go

```diff
@@ -220,16 +222,21 @@ func (fri *FakeRefineryInstance) runDecider(opts CmdLineOptions, nodeIndex int,
 						span3.End()
 						continue
 					}
-					trace, err := fri.Store.GetTrace(getCtx, status.TraceID)
+					traces, err := fri.Store.GetTraces(getCtx, status.TraceID)
 					if err != nil {
 						otelutil.AddException(span3, err)
 					}
+					if len(traces) == 0 {
+						otelutil.AddSpanField(span3, "decision", "no trace")
```

Comment:
Is "decision" the right field here? That seems weird, but maybe it's fine.

Reply:
I changed it to `not_found`